### PR TITLE
fix issue 15336, opIn undocumented for JSONValue

### PR DESCRIPTION
**Pull request by Blumerline**
- document opIn
- + fix some DDoc sections

### CODE REVIEW

**Comment by Blumerline** [comment]
DAutoTest failure is a fake positive.

**Comment by Blumerline** [comment]
fix for issue 15336 is updated.

File: std/json.d [hunk 1]
```diff
@@ -17,13 +17,12 @@ Synopsis:
 
     //check a type
     long x;
-    if (j["code"].type() == JSON_TYPE.INTEGER)
+    if ("code" in j)
```
Comment (AndrejMitrovic) [inline std/json.d]: Seems like you could possibly use:

```D
if (auto code = "code" in j)
{
    if (code.type == JSON_TYPE.INTEGER)
        x = code.integer;
    else
        x = code.str.to!int;
}
```

**Comment by Blumerline** [comment]
fix for issue 15336 is updated.

**Pull request closed without merging**